commands/tankDrive.py

```diff
@@ -1,12 +1,13 @@
 import commands2
 import logging
 from typing import Callable
+import wpilib
 log = logging.getLogger(__name__)
 class TankDrive(commands2.CommandBase):
     '''
     Command for converting joystick input to drive train output
     '''
-    def __init__(self, left: Callable[[], float], right: Callable[[], float], driveTrain):
+    def __init__(self, left: Callable[[], float], right: Callable[[], float], creeperMode: Callable[[], bool], driveTrain):
```

Comment:
```suggestion
    def __init__(self, left: Callable[[], float], right: Callable[[], float], creeperMode: Callable[[], bool], driveTrain):
    #TODO replace crepper with command instead of bypassing commands
```